packages/palette-docs/content/docs/tokens/Color.mdx

```diff
@@ -6,15 +6,6 @@ See [Notion](https://www.notion.so/artsy/Color-a0c24896daf8433d9409aee2146ac267)
 for more detailed spec.
 
 <Box>
-  {Object.entries(themeProps.colors).map(([colorCode, hex], index) => {
-    return (
-      <Box mb={3} key={index}>
-        <Sans size="3" weight="medium">
-          {colorCode}
-        </Sans>
-        <Sans size="3">{hex}</Sans>
-        <Box width="100%" height={5} bg={colorCode} />
-      </Box>
-    )
-  })}
+  <ColorComponent color="black100" hex="#000" />
+  <ColorComponent color="black80" hex="#333" />
```

Comment:
Why would we list only two colors? 

Shouldn't we match the Zeplin spec so that the docs are useful? (Two black colors also isn't very pretty, compared to before.)

@owendodd  - we can restructure the docs around the the new Primary, Secondary, and Deprecated sections; we don't need to throw everything out.

Reply:
thanks @damassi - thought you were out sick today or would've requested your review! yes, this will show all of the colors, I just figured I'd leave it to Owen to update using admin after merging

Reply:
We should probably think about this in a more dynamic way -- e.g., grouping things behind the three categories and then iterating over a JSON object (vs a manual list)

Reply:
And the intent here definitely isn't to scrap everything; I didn't think we could easily separate out the colors into sections with the previous `map` implementation. But open to ideas!

Reply:
```
const colors = [
  {
    hex: "FF0000",
    category: "primary"
  },
  ... 
]
```
@owendodd  - would you mind creating a design in Zeplin that we could work from? In the meantime we can probably put this PR on hold.

Reply:
I'm not too concerned about having group the `primary`, `secondary`, `depreciated` colors more manually here as we will want to add guidelines for the usage of each color alongside the rendered blocks anyways, which seems a little tricky to do while iterating through them all. 

In my mind the idea solution would be be something where I can quickly input a color name (e.g. black30), which would return a block that includes the color name, hex code, and a preview of the color. Then I could easily add text below that outlines the useage guidelines for that color and group accordingly under a 'Primary', 'Secondary' or 'Depreciated' header.

Reply:
👍 that makes sense. 

Y'all will need to update the [Global Components](https://github.com/artsy/palette/blob/master/packages/palette-docs/src/components/GlobalComponents.jsx#L75) file as well.

Reply:
ah cool, thanks @damassi! learning my way around the new docs site